### PR TITLE
media-libs/glm: Version bump, fixes #553156 and #538294

### PR DESCRIPTION
**Pull request by marbre**
@mgorny In https://bugs.gentoo.org/show_bug.cgi?id=538294 you suggested to enable the CMake tests. This is done within this version bumped ebuild. The version is bumped to 0.9.7.1.
If desired I can enable the CMake build system also for lower versions or also add 0.9.6.3, as originally reported in https://bugs.gentoo.org/show_bug.cgi?id=553156.

Further, I added `virtual/opengl` to `RDEPEND`.

### CODE REVIEW

File: media-libs/glm/glm-0.9.7.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="OpenGL Mathematics"
+HOMEPAGE="http://glm.g-truc.net/"
+SRC_URI="https://github.com/g-truc/glm/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
```
Comment (hasufell) [inline media-libs/glm/glm-0.9.7.1.ebuild]:
```
GLM can be distributed and/or modified under the terms of either
a) The Happy Bunny License, or b) the MIT License.
```
https://github.com/g-truc/glm/blob/master/copying.txt#L4

we should leave the choice to the user

Comment (marbre) [inline media-libs/glm/glm-0.9.7.1.ebuild]: I agree. Added "The Happy Bunny" license as HappyBunny and adjusted the ebuild.

**Comment by hasufell** [comment]
the license commit should have the commit message
```
licenses: add HappyBunny license
```

**Comment by marbre** [comment]
@hasufell Done.

**Comment by mgorny** [comment]
@marbre, do you want to become proxy-maintainer for this package? If so, please add yourself and proxy-maintainers to metadata.xml.

**Comment by marbre** [comment]
@mgorny Unfortunately, I must reject. Currently I have enough to do with jupyter, arrayfire and cuda ebuilds in the science overlay and I already proxy-maintain mendeleydesktop.

Only requested this bump as the CMake configuration provided with newer glm versions is required by arrayfire/forge. A pull requested seemed more useful than adding a bumped version to the science overlay.
Anyway, maybe adding a herd to the package would be a good idea. I hopefully join the sci herd as developer in near future.

Beside, I adjusted the metadata.xml to point to the correct upstream repository.

**Comment by mgorny** [comment]
@gentoo/sci, how would you feel about taking this?

**Comment by jlec** [comment]
Merged.

I am sorry, but had some commits myself, so did a rebase and squashed the merge commit. Next time I will do better.

**Pull request closed without merging**